konfera/tests/test_models.py

```diff
@@ -409,3 +409,17 @@ def test_status(self):
         tt.date_from = now - datetime.timedelta(days=3)
         tt.date_to = now - datetime.timedelta(days=1)
         self.assertEquals(tt.status, TicketType.STATUSES[TicketType.EXPIRED])
+
+
+class OrganizerTest(TestCase):
+
+    def setUp(self):
+        self.first_organizer = models.Organizer(title='Mysterious Organizer', street='1 Up street', city='Big City',
+                                                company_id='123',
+                                                about_us="World famous yet unknown conference organizer.")
+        self.noname_organizer = models.Organizer(street='2 Random', city='Small City')
+
+    def test_save(self):
+        self.assertEquals(self.first_organizer.save(), None)
+        self.assertEquals(str(self.first_organizer), 'Mysterious Organizer')
```

Comment:
The test's name is `test_save` so we should test only the `.save()` method here. Please move this (`str`) assertion to a new test.